node/src/components/chain_synchronizer.rs

```diff
@@ -276,7 +277,7 @@ impl ChainSynchronizer {
                     PublicKey::System,
                 );
 
-                self.execute_block(effect_builder, initial_pre_state, finalized_block)
+                self.execute_block(effect_builder, None, initial_pre_state, finalized_block)
```

Comment:
We should rename that to `execute_immediate_switch_block`, since the chain synchronizer also executes regular blocks elsewhere.

Reply:
Addressed in [3f0bfa0](https://github.com/casper-network/casper-node/pull/2785/commits/3f0bfa034550037b59232d8bef9b78cf2f007f4e).